cjworkbench/tests/i18n/test_trans.py

```diff
@@ -72,23 +116,32 @@ def test_trans_params(self):
             "Hello you there {c}!",
         )
 
-    # Tests that a programmer will get an exception when including a numeric variable in the message
+    # Tests that a programmer can include a numeric variable in the message
     def test_format_invalid_default(self):
-        with self.assertRaises(InvalidICUParameters):
+        self.assertEqual(
             trans_html(
                 default_locale,
                 mock_message_id,
                 default="Hello {a} {0} {b}",
                 parameters={"a": "you", "0": "!", "b": "2"},
             ),
+            "Hello you ! 2",
+        )
 
     # Tests that a translator can't break our system by including a numeric variable in the message
     def test_format_invalid_message(self):
-        self.assertEqual(
+        self.assertTrue(
```

Comment:
... this should be a `self.assertEqual()`.

What is it equal to? I'm in suspense.